src/AbstractOperations/metric_field_reductions.jl

```diff
@@ -23,8 +23,29 @@ reduction_grid_metric(dims) = dims === tuple(1)  ? Δx :
 ##### Metric reductions
 #####
 
-struct Averaging <: AbstractReducing end
+struct Averaging{V} <: AbstractReducing
+    volume :: V
+end
```

Comment:
just curious: why the parameter V needed to be introduced; I was trying to see where it's used and I'm missing it?